### ISSUE TITLE
Config elements disappearing at random

### ISSUE THREAD

**Issue by snwflake** [ElvUI-MoP/ElvUI-5.4.8#70]
```
3x attempt to compare number with string
[C]: ?
...es\Ace3\12AceGUI-3.0-41\widgets\AceGUIWidget-DropDown.lua:629: in function `SetList'
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0-59.lua:1357: in function <...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:1179>
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0-59.lua:1206: in function <...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:1179>
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0-59.lua:1762: in function `FeedGroup'
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0-59.lua:1687: in function <...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:1671>
(tail call): ?
[C]: ?
[string "safecall Dispatcher[3]"]:9: in function <[string "safecall Dispatcher[3]"]:5>
(tail call): ?
...I_OptionsUI\Libraries\Ace3\AceGUI-3.0\AceGUI-3.0-41.lua:335: in function `Fire'
...ce3\8AceGUI-3.0-41\widgets\AceGUIContainer-TreeGroup.lua:518: in function `Select'
...
[string "safecall Dispatcher[3]"]:9: in function <[string "safecall Dispatcher[3]"]:5>
(tail call): ?
...I_OptionsUI\Libraries\Ace3\AceGUI-3.0\AceGUI-3.0-41.lua:335: in function `Fire'
...es\Ace3\12AceGUI-3.0-41\widgets\AceGUIWidget-DropDown.lua:442: in function <...es\Ace3\AceGUI-3.0\widgets\AceGUIWidget-DropDown.lua:433>
(tail call): ?
[C]: ?
[string "safecall Dispatcher[3]"]:9: in function <[string "safecall Dispatcher[3]"]:5>
(tail call): ?
...I_OptionsUI\Libraries\Ace3\AceGUI-3.0\AceGUI-3.0-41.lua:335: in function `Fire'
...3\6AceGUI-3.0-41\widgets\AceGUIWidget-DropDown-Items.lua:351: in function <...3\AceGUI-3.0\widgets\AceGUIWidget-DropDown-Items.lua:341>

Locals:
nil
```

![](https://i.imgur.com/ZDzatud.jpeg)

**Comment by snwflake** [comment]
Not the slightest idea how to reproduce

Edit: Way to reproduce ~

Filter > Use Dropdown, select RaidDebuffs > Use Dropdown, select None
above error pops

**Comment by snwflake** [comment]
fixed with 5b4e97ff
